frontend/www/js/omegaup/components/arena/Clarification.vue

```diff
@@ -95,8 +131,21 @@ export default class ArenaClarificationForm extends Vue {
     },
   ];
 
+  get responseText(): string {
+    const [response] = this.responses.filter(
```

Comment:
qué tal mejor usar [`Array.find()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)?

```suggestion
    const response = this.responses.find(
```

Reply:
Listo